src/components/Game/Game.tsx

```diff
@@ -17,7 +17,7 @@ import { Button } from "../Button";
 import { PlayerHand } from "../PlayerHand";
```

Comment:
I'm a bit confused, we have a path /game and we have a Game component, but it's not a page it's inside components